tests/test_entry_resource.py

```diff
@@ -12,7 +14,26 @@ def test_content_type(self, response):
         assert response.headers['content-type'] == 'application/json'
 
     @pytest.mark.xfail
```

Comment:
yes can remove xfail now because the entry resource is deployed completely in beta environment.